certbot/constants.py

```diff
@@ -156,6 +156,13 @@
 ACCOUNTS_DIR = "accounts"
 """Directory where all accounts are saved."""
 
+LE_REUSE_SERVERS = {
+    'acme-v02.api.letsencrypt.org/directory': 'acme-v01.api.letsencrypt.org/directory',
```

Comment:
This feature is in staging now but won't be in prod until after our release next week. Would you prefer:

1. We temporarily remove this line and merge this PR, enabling account reuse in staging (which will affect people using flags like `--dry-run`).
2. We delay this entire PR until it's enabled in prod as well.

I prefer the former.

Reply:
yes, former is great

Reply:
nice to have staging test account reuse anyway